Trading/Exchange/bybit.py

```diff
@@ -56,10 +62,9 @@ def set_sandbox_mode(self, is_sandboxed):
             self.client.urls['api'] = 'https://testnet.bybit.com'
         self.client.setSandboxMode(is_sandboxed)
 
-    async def get_symbol_open_positions(self, symbol: str) -> dict:
-        return self._cleanup_position_dict(
-            (await self.client.private_get_position_list(
-                {"symbol": self.get_exchange_pair(symbol)}))['result'])
+    async def get_symbol_open_positions(self, symbol: str) -> list:
+        return [self.cleanup_position_dict((await self.client.private_get_position_list(
+            {"symbol": self.get_exchange_pair(symbol)}))['result'])]
 
     async def set_symbol_leverage(self, symbol: str, leverage: int):
         # TODO returned error during implementation tests
```

Comment:
still erroring ?

Reply:
Yes, I didnt investigate much about this issue. That's the reason of the `TODO`.